development-vm/Vagrantfile

```diff
@@ -105,6 +105,8 @@ Vagrant.configure("2") do |config|
     config.vm.synced_folder "../..", "/var/govuk", :type => :nfs
   end
 
+  config.vm.provision "file", source: "~/.gitconfig", destination: "~/.gitconfig"
```

Comment:
I wonder what this does if the file isn't present?

Reply:
Good point. I imagine it would error. But I'll raise a PR to update [the 'Get Started' page](https://docs.publishing.service.gov.uk/manual/get-started.html) so that it instructs people to set up their global git config on their host machine, before they try doing anything in the VM.

Reply:
Raised https://github.com/alphagov/govuk-developer-docs/pull/1783